jscomp/test/Import.res

```diff
@@ -0,0 +1,17 @@
+let eachIntAsync = async (list: list<int>, f: int => unit) => {
```

Comment:
With the current checks turned off, if you remove `async` here it does not give an error.